domain/src/test/kotlin/com/antosik/benchproject/domain/popular/movies/usecase/GetPopularMoviesUseCaseTest.kt

```diff
@@ -10,12 +10,7 @@ import org.junit.jupiter.api.Test
 
 internal class GetPopularMoviesUseCaseTest {
 
-    val movieListModel = listOf(
-        MovieModel(10001, "Test1", 5.4, "2022-06-23", "https:/something1.com", false),
-        MovieModel(10002, "Test2", 6.4, "2022-06-22", "https:/something2.com", false),
-        MovieModel(10003, "Test3", 3.4, "2022-06-21", "https:/something3.com", false)
-    )
-
+    val movieListModel = listOf<MovieModel>(mockk())
```

Comment:
You can use here `val movieListModel: List<MovieModel> = mockk()` :) simpler and cleaner if You want to only mock the list

Reply:
Yes, it looks better 😄

Reply:
Done